app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -47,7 +47,7 @@ If you're coming to the UK from Ukraine, [check what you need to do before you t
 
 ### Get an international relocation payment (IRP) worth £10,000
```

Comment:
@laurahermionetennant92 are there people still applying who could be eligible? For e.g. their course starts before 31 August? If not then the H2 may be misleading as they can't get one. It may be better to just have International location repayment (IRP) and then refer to the content below in the past tense?

Or do we even need to have this section at all any more?

Reply:
Ukrainian teachers will still be eligible, it's trainees who won't be – is that what you mean?

Reply:
@laurahermionetennant92 I've re-read this section and I was still a little confused. I think it could be good to call out that this is for teachers to avoid any confusion as the page covers teachers and trainees. Realise it's mentioned within the copy but I think calling it out in this H2 Get an international relocation payment (IRP) worth £10,000 so maybe something like Get an international relocation payment (IRP) worth £10,000 **if you're a teacher** (or something better!)

And I also think it could be worth making it clearer in this bit To be eligible for the IRP, you must be coming to England to teach languages or physics in:

As a trainee you could see yourself as coming to teach so again might be worth reiterating it's for teachers: To be eligible for the IRP, you must **be a teacher** coming to England to teach languages or physics in: